packages/zoe/src/zoe.js

```diff
@@ -615,6 +627,10 @@ const makeZoe = (additionalEndowments = {}) => {
        */
       getInstance: instanceTable.get,
 
+      /** Get a notificationRecord (see @agoric/notify) for the offer. */
+      getOfferNotifications: offerHandle =>
+        offerTable.get(offerHandle).notifier.getUpdateSince(),
```

Comment:
Since the only thing about an offer that varies over time is its allocation, this does sensibly provide only a stream of allocation change notifications. However, its name suggests it provides offer records.

Reply:
resolved after discussion.